### DISCUSSION TITLE
Can I use the Individual license in production as a single developer?

### DISCUSSION THREAD

**Discussion by Maskoe** [datalust/seq-tickets#2130] [Help] [answered]
Can I take the docker image, host it somewhere, send my logs and look at it with the Individual license?

I saw a [similiar thread](https://github.com/datalust/seq-tickets/discussions/1807) that just linked the EULA.  
Im looking for an answer that a real human can understand.

Its not on my machine and its using the docker image so I think that counts as a "server install", which is not included in the free, individual license?

**Comment by nblumhardt** [comment, marked as answer]
Hi Mirko, thanks for reaching out, I see my reply in the other thread could have been clearer :-)

There's no production vs. non-production distinction made in the license. As long as your use meets the "Individual" requirements (e.g. you're the person who is both installing and will use the instance, it's not for redistribution, etc.) you can use it for production diagnostics.

**Reply by nblumhardt** [reply]
Sorry, accidental "Close" button press!

Was adding: the "server install" wording is used to essentially mean "Seq install" - an earlier source of confusion around this was whether Seq licensing limited the number of applications that could log to Seq, which it doesn't, so this term helps clear that up. Thanks for pointing out that there's still some room for an additional FAQ around this 👍 

Hope this helps,
Nick

**Comment by k3davis** [comment]
Perhaps this issue is addressed elsewhere but in my opinion the pricing page is ambiguous on this point. It says a single user can use it in production, but "on a developer workstation" which most would not define as "in production". A single user server install is not mentioned anywhere as far as I can tell, and I find conflicting interpretations of that elsewhere on the web.

**Reply by nblumhardt** [reply]
Thanks for the note, @k3davis. The best option might be for us to put a EULA link more prominently on the pricing page, since it's the ultimate source for this info (and covers the points we're concerned about confusion around, such as the no bundling/no redistribution requirements).
